src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,140 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.25.0.dev3 (01/29/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Remove the generate-pants-ini goal. (#8993)
+  `PR #8993 <https://github.com/pantsbuild/pants/pull/8993>`_
+
+* Deprecate `--fast` option (#8970)
+  `PR #8970 <https://github.com/pantsbuild/pants/pull/8970>`_
+
+* Switch to dot idea project type (#8745)
+  `PR #8745 <https://github.com/pantsbuild/pants/pull/8745>`_
+
+* Deprecate defaulting to `--transitive` (#8955)
+  `PR #8955 <https://github.com/pantsbuild/pants/pull/8955>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow excludes in BUILD files via new `!` syntax (#9025)
```

Comment:
This is probably more of an API change.

Reply:
I disagree. This is a new feature and the API change will be deprecating `globs` in favor of this style. Here, we're simply adding a new way to define things and not changing any prior APIs.